### DISCUSSION TITLE
Ender 3 v2 precompiled v1.3.2 not showing files

### DISCUSSION THREAD

**Discussion by bring42** [Jyers/Marlin#691] [Q&A] [answered]
Hi! I flashed the precompiled v1.3.2 E3V2-ManualMesh-3x3-v4.2.2 this morning after being on an old version since February. Everything went smooth and things seemed to work (mesh leveling and so on) until I was going to start a print. It says that media was inserted, but the files list is empty. I have tried re-flashing, formated the sd-card, other gcode files but nothin seems to work. Is this a known issue or is there something I might have missed? 

Installed the wrong firmware first (4.2.7) but quickly noticed it was not able to move the motors, could that have caused the issue? 

edit: gcode files, not stl files

**Comment by smooreace** [comment]
Installing the 4.2.7  firmware wont hurt anything. Just install the correct
version afterward and everything should be fine.

Just going by what youre are saying, you formatted the SDcard... did you
put the code files back on the card afterward? And to be extra clear, you
cant print an STL file. If contains zero usable data for a printer

**Reply by bring42** [reply]
Ya I thought that would not be an issue, and like you say everything was fine after flashing the right version. (other than this file problem) 

Ohh, my bad! I meant gcode files, saved directly to the card after slicing in Cura. 
A rough timeline would be: format -> install firmware -> format -> save prints (.gcode)

**Comment by Jyers** [comment, marked as answer]
Try resetting and storing from the control menu to clear eeprom, old eeprom can cause all sorts of issues.

**Reply by bring42** [reply]
Yes! Now its working, thanks!
